modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -77,6 +79,19 @@ public class UpgradeCatchAllCheck {
 		_commerceAccountHelper.getCurrentCommerceAccount(null);
 	}
 
+	public void method(
+		HttpServletRequest httpServletRequest, String name) {
+		CookieUtil.get(httpServletRequest, "test123");
```

Comment:
@MicaelleSilvaa, when a method signature is declared in 2 or more lines then we need to separate it from the method's content using a blank line, please see an example [here](https://github.com/natocesarrego/liferay-portal/blob/9f0279d3781597b87b32ec8350a9236a8da778a4/modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava#L64-L67).

Reply:
Resolved